app/livechat/server/config.js

```diff
@@ -329,16 +329,6 @@ Meteor.startup(function() {
 		i18nLabel: 'Continuous_sound_notifications_for_new_livechat_room',
 	});
 
-	settings.add('Livechat_videocall_enabled', false, {
```

Comment:
You've removed this setting here but I can still find references to this within the codebase. Could you please check this and modify those references accordingly?

Secondly, we'll also need another migration for this setting. Users who already have this and jitsi setting on, should have the `Jitsi` option selected within our new setting i.e. `Livechat_call_provider`